internal/rbd/encryption.go

```diff
@@ -135,8 +135,9 @@ func (ri *rbdImage) copyEncryptionConfig(cp *rbdImage) error {
 		return fmt.Errorf("failed to fetch passphrase for %q: %w",
 			ri, err)
 	}
-
-	cp.encryption, err = util.NewVolumeEncryption(ri.encryption.GetID(), ri.encryption.KMS)
+	if !copyOnlyPassphrase {
```

Comment:
add empty lines around this if-block to increase readability?

Move the error check and `SetDEKStore` inside the if-statement too?